src/Authentication/AuthenticatorInterface.php

```diff
@@ -45,9 +45,11 @@ public function login(Authenticatable $user);
      *
      * @see https://codeigniter4.github.io/CodeIgniter4/extending/authentication.html
      *
+     * @param int|object|string $userId
```

Comment:
What would an `object` even be here? I propose dropping that.

Reply:
Since it is an interface, I think it is better to include the possibilities.
I assumed a user who uses a value object.

Reply:
I'd actually argue the opposite: since it is an interface we should keep it tight. Adding the possibility for an `object` means everyone's authentication handler is going to have to check for and deal with an `object` with an unspecified structure. If someone wants to pass an object they could use a `Stringable` and still fit within this interface.

Reply:
Okay, this is Auth system. `int|string` is enough for User ID for authentication.
Fixed.